ios/Lunes.xcodeproj/project.pbxproj

```diff
@@ -634,7 +634,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
Why have you changes this?

Reply:
just by mistake. this is only for simulators needed on m1. undone